docs/concepts/macros/macro_variables.md

```diff
@@ -100,3 +100,22 @@ Other macro variables:
     * 'creating' - The model tables are being created.
     * 'evaluating' - The models' logic is being evaluated.
 * @gateway - A string value that represents the name of the selected [gateway](../../guides/connections.md).
+
+## User-defined Variables
+
+Users can also pass custom variables in [configuration](../../reference/configuration.md#variables). To access variable values the `@var` macro should be used in model definitions. For example:
+
+```sql linenums="1"
+SELECT *
+FROM table
+WHERE some_value = @var('<var_name>')
```

Comment:
why use `@var` and not just the variable name?

Reply:
More importantly though, having a function allows you to specify a default value in the second argument.

Reply:
After additional conversation we decided to support both styles